eslint-plugin-relay.js

```diff
@@ -479,14 +483,46 @@ module.exports.rules = {
             FragmentSpread(spreadNode) {
               const m =
                 spreadNode.name &&
-                spreadNode.name.value.match(/^([a-z0-9]+)_/i);
-              if (!m) {
+                spreadNode.name.value.match(/^([a-z0-9]+)_([a-z0-9]+)/i);
```

Comment:
The prop name regexp should probably include `_` to match `Foo_bar_zomg` as the prop name `bar_zomg`. I think that's the logic we use elsewhere.